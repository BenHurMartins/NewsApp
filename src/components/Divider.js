import React from 'react';
import {StyleSheet, View} from 'react-native';
import {Colors, Mixins} from '../styles';

const Divider = props => {
  return <View style={[styles.container, props.style]} />;
};

export default Divider;

const styles = StyleSheet.create({
  container: {
    height: 1,
    backgroundColor: Colors.PRIMARY,
    width: Mixins.DEVICE_WIDTH,
  },
});
