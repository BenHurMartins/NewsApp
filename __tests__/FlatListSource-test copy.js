import 'react-native';
import React from 'react';
import Divider from '../src/components/Divider';
import renderer from 'react-test-renderer';

describe('Divider component', () => {
  it('Divider renders correctly', () => {
    renderer.create(<Divider />);
  });
  it('Divider snapshot test', () => {
    const tree = renderer.create(<Divider />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
