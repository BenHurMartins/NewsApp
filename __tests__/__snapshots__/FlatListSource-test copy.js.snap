// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider component Divider snapshot test 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#65a1dd",
        "height": 1,
        "width": 750,
      },
      undefined,
    ]
  }
/>
`;
